code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -110,6 +110,8 @@
 							I.mechassist()
 						if ("mechanical")
 							I.robotize()
+						if ("removed")
+							qdel(I)
```

Comment:
I think you need to call the organ's removed proc, unless its Destroy() does that already?

Reply:
Destroy() Does most of what removed does